R/install-local.r

```diff
@@ -46,3 +46,13 @@ remote_metadata.local_remote <- function(x, bundle = NULL, source = NULL) {
     RemoteSha = if (uses_git(x$path)) git_sha1(path = x$path)
   )
 }
+
+#' @export
+remote_metadata.package <- function(x, bundle = NULL, source = NULL) {
```

Comment:
Actually I am wrong, just looked at the code for `pkg_source` https://github.com/hadley/devtools/blob/81dd313e1d143a3bdaaa2e1f09c887801db66694/R/session-info.r#L156-L159